app/application.js

```diff
@@ -293,8 +264,9 @@ export class DesktopApplication {
       event.preventDefault();
     });
   }
+}
 
-  openDevTools() {
```

Comment:
We should hang on to this because sometimes you really want the dev console to open up by default when you're debugging ;) I use this a lot.